### ISSUE TITLE
Missing accessibility

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#55]
Hello! I noticed that the study group map is missing some accessibility features. Could you please add them? It would be great to have a more accessible map for everyone. Thank you!
